x/ibctransfer/whitelisting_test.go

```diff
@@ -0,0 +1,52 @@
+package ibctransfer
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	transfertypes "github.com/cosmos/cosmos-sdk/x/ibc/applications/transfer/types"
+	channeltypes "github.com/cosmos/cosmos-sdk/x/ibc/core/04-channel/types"
+	"github.com/stretchr/testify/require"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+func TestIsRecvPacketAllowed(t *testing.T) {
```

Comment:
I think we could have more tests to cover all whitelisting functions, ideally success/failure case for each one

Reply:
Yup, will do once we integrate the whitelist keeper.